pkg/web/templates/counterparty.html

```diff
@@ -1,10 +1,10 @@
 {{ template "base" . }}
 {{ define "content" }}
   <section class="mx-8 py-14">
-    <section class="flex gap-x-6 mb-12">
+    <section class="md:flex md:items-center md:gap-x-6 mb-12">
```

Comment:
This updates the counterparty page so that the button appears below the heading on smaller screens. The change was made to be consistent with other pages.